distributions/Lakka/options

```diff
@@ -284,6 +284,7 @@ DISTRO_SRC="http://sources.libreelec.tv/$LIBREELEC_VERSION"
 " xrick "\
 " yabasanshiro "\
 " yabause "\
+" jumpnbump "\
```

Comment:
May you try to keep the alphabetic order ;)

Thank you.

Reply:
Arf didn't noticed... I'll change that